ReceiveVoiceCall/ReceiveVoiceCall.ino

```diff
@@ -25,7 +25,11 @@
 #include <GSM.h>
 
 // PIN Number
-#define PINNUMBER "1111"
+#define PIN "1111"
+
+int redPin = 10;
+int bluePin = 11;
+int greenPin = 12;
```

Comment:
General comment about this file: do I see it correctly that you have *ReceiveVoiceCall* and *arduinoClient* folders? I would change it to *arduinoClientReceive* and *arduinoClientCall*.